app/experimenter/templates/experiments/rapid.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+
+{% block main %}
+Rapid Experiments
+{% endblock %}
+
+{% block extrascripts %}
+{% endblock %}
```

Comment:
nittiest of nits: EOF new line

Reply:
Yeah we don't have any way to autoformat the templates, I've been looking....